python/src/main/python/drivers/piglit-worker.py

```diff
@@ -305,32 +320,33 @@ def main():
 
     gfuzz_common.tool_on_path('shader_runner_gles3')
     gfuzz_common.tool_on_path('glxinfo')
+    gfuzz_common.tool_on_path('catchsegv')
```

Comment:
`catchsegv` (and `glxinfo` actually) are Linux only. It would be ideal for this script to work (in theory) on Mac, Linux, and Windows. I.e. if `glxinfo` is not found, that is fine (which I believe you pretty much already have, apart from the above) and if `catchsegv` is not found, then it should not be used and `gfuzz_common.run` should be used in that case also. However, perhaps this should be addressed in a follow-up PR.

Reply:
Done.